dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/test/TestConnector.java

```diff
@@ -22,15 +22,14 @@
 import com.google.edwmigration.dumper.application.dumper.connector.Connector;
 import com.google.edwmigration.dumper.application.dumper.connector.ConnectorProperty;
 import com.google.edwmigration.dumper.application.dumper.connector.MetadataConnector;
-import com.google.edwmigration.dumper.application.dumper.handle.AbstractHandle;
 import com.google.edwmigration.dumper.application.dumper.handle.Handle;
 import com.google.edwmigration.dumper.application.dumper.task.Task;
 import java.util.List;
 import javax.annotation.Nonnull;
 
 @AutoService({Connector.class, MetadataConnector.class})
 public class TestConnector extends AbstractConnector implements MetadataConnector {
-  private static final Handle DUMMY_HANDLE = new AbstractHandle() {};
+  private static final Handle DUMMY_HANDLE = () -> {};
```

Comment:
This change looks good, I like it.

Reply:
This is not affected by changes in `AbstractHandle`, it would have to be a change in `Handle`.

This is also a test fake, only for cases where we need to pass something that resembles a handle for the code to run. If the value of the handle was relevant, we should use a real handle instead of a dummy class or `() -> {}`.